### ISSUE TITLE
Play should be updated to Lessons

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#603]
As per new mocks, Topic Play Tab has been updated to Topic Lessons Tab,

<img width="363" alt="Screenshot 2020-01-03 at 12 11 29 PM" src="https://user-images.githubusercontent.com/9396084/71710439-3fa89500-2e22-11ea-83c9-a5564e000459.png">

So, all files related to that needs to be updated like package name, file names, test cases names, xml files names, variable names, strings.xml, drawables, etc.

In short refactor `Play` with `Lessons`.

**Comment by rt4914** [comment]
Assigning this to @prarabdhgarg
